### PR TITLE
Rename columns

### PR DESCRIPTION
**Pull request by stefpiatek**
Here's a quick go at adding in how to rename columns. Didn't feel like it was worth adding it as a new commonly used verbs to dplyr so just tagged it onto the end of the select, as they're already dealing with columns there. Let me know what you think. 

Also figure I may as well try and get this to count to hacktoberfest when you're happy with it. Looks like you'd need to either tag the PR or the repository as a whole if that's not too much hassle!

Closes swcarpentry/r-novice-gapminder#660

### CODE REVIEW

File: _episodes_rmd/13-dplyr.Rmd [hunk 1]
```diff
@@ -98,6 +98,18 @@ gets that from the previous pipe. **Fun Fact**: There is a good chance you have
 encountered pipes before in the shell. In R, a pipe symbol is `%>%` while in the
 shell it is `|` but the concept is the same!
 
+> ## Tip: Renaming dataframe columns in dplyr
+>
+> In Chapter 4 we covered how you can rename columns with base R by assigning a value to the output of the `names` function. Just like select, this is a bit cumbersome, but thankfully dplyr has a `rename` function.
```
Comment (naupaka) [inline _episodes_rmd/13-dplyr.Rmd:103]: For Carpentries R style, we prefer functions to be followed by parentheses for distinguishing them from other objects (e.g. `names()`, `select()`, and `rename()`)

File: _episodes_rmd/04-data-structures-part1.Rmd [hunk 2]
```diff
@@ -376,6 +376,26 @@ These make sense. But what about
 str(cats$coat)
 ```
 
+> ## Tip: Renaming dataframe columns
+>
+> Dataframes have column names we can access them like we did with vectors
```
Comment (naupaka) [inline _episodes_rmd/04-data-structures-part1.Rmd:381]: Grammar: "Dataframes have column names we can access them like we did with vectors" -> "Data frames have column names, which can be accessed with the `names()` function."

File: _episodes_rmd/04-data-structures-part1.Rmd [hunk 3]
```diff
@@ -376,6 +376,26 @@ These make sense. But what about
 str(cats$coat)
 ```
 
+> ## Tip: Renaming dataframe columns
```
Comment (naupaka) [inline _episodes_rmd/04-data-structures-part1.Rmd:379]: dataframe -> data frame

File: _episodes_rmd/04-data-structures-part1.Rmd [hunk 4]
```diff
@@ -376,6 +376,26 @@ These make sense. But what about
...
+>
+> Dataframes have column names we can access them like we did with vectors
+>
+> ```{r}
+> names(cats)
+> ```
+>
+> If you want to rename the second column you can assign the new name to the second element of names
```
Comment (naupaka) [inline _episodes_rmd/04-data-structures-part1.Rmd:387]: Grammar: "If you want to rename the second column you can assign the new name to the second element of names" -> "If you want to rename the second column, you can assign a new name to the second element of `names(cats)`."

File: _episodes_rmd/13-dplyr.Rmd [hunk 5]
```diff
@@ -98,6 +98,18 @@ gets that from the previous pipe. **Fun Fact**: There is a good chance you have
 encountered pipes before in the shell. In R, a pipe symbol is `%>%` while in the
 shell it is `|` but the concept is the same!
 
+> ## Tip: Renaming dataframe columns in dplyr
+>
+> In Chapter 4 we covered how you can rename columns with base R by assigning a value to the output of the `names` function. Just like select, this is a bit cumbersome, but thankfully dplyr has a `rename` function. 
+>
+> Within a pipeline, it's used as `rename(new_name = old_name)`. For example we may want to rename the gdpPercap column name from our select statement above.
```
Comment (naupaka) [inline _episodes_rmd/13-dplyr.Rmd:105]: "it's used as" -> "the syntax is"

Comment (naupaka) [reply _episodes_rmd/13-dplyr.Rmd:105]: Grammar: "For example we may want to rename the gdpPercap column name from our select statement above." -> "For example, we may want to rename the `gdpPercap` column name from our `select()` statement above."

**Review by naupaka — CHANGES_REQUESTED** [review]
Thanks for the PR @stefpiatek! I've made some suggestions that mostly have to do with style and grammar issues. I am not sure if the Carpentries are participating in Hacktober? @jcoliver may know more. I know there were some issues with lots of spam across a number of the lesson repos.

**Comment by stefpiatek** [comment]
Thanks for the feedback - should all be addressed now but let me know if you spot anything else that needs changing. 

No worries if hacktoberfest isn't something that the carpentries are doing.

File: _episodes_rmd/13-dplyr.Rmd [hunk 6]
```diff
@@ -100,9 +100,11 @@ shell it is `|` but the concept is the same!
 
 > ## Tip: Renaming dataframe columns in dplyr
 >
-> In Chapter 4 we covered how you can rename columns with base R by assigning a value to the output of the `names` function. Just like select, this is a bit cumbersome, but thankfully dplyr has a `rename` function. 
+> In Chapter 4 we covered how you can rename columns with base R by assigning a value to the output of the `names()` function.
+> Just like select, this is a bit cumbersome, but thankfully dplyr has a `rename` function.
```
Comment (naupaka) [inline _episodes_rmd/13-dplyr.Rmd:104]: Please add parentheses after rename

**Comment by naupaka** [comment]
Need to add parentheses in one place and then it should be good to merge.

@fmichonneau what's the current status of Carpentries and Hacktober?

File: _episodes_rmd/13-dplyr.Rmd (hunk 6 above)
Comment (stefpiatek) [reply _episodes_rmd/13-dplyr.Rmd:104]: ah so close, good spot

**Review by naupaka — APPROVED** [review]

**Comment by naupaka** [comment]
Looks good to me. I will wait a few days to see if @fmichonneau has an update re: Hacktober, otherwise this is ready to merge.

**Comment by fmichonneau** [comment]
@stefpiatek I added the `hacktoberfest-accepted` which should be enough for your contribution to count. Could you please check your dashboard and get back to us if you don't see your PR there?

**Comment by stefpiatek** [comment]
> @stefpiatek I added the `hacktoberfest-accepted` which should be enough for your contribution to count. Could you please check your dashboard and get back to us if you don't see your PR there?

Thanks @fmichonneau, yep it's in my dashboard - thanks!

**Comment by naupaka** [comment]
Thanks all!

**Pull request merged**